src/pages/Background/redux-slices/index.ts

```diff
@@ -62,7 +62,7 @@ const reduxCache = (store) => (next) => (action) => {
 };
 
 export const initializeStore = (
-  preloadedState,
+  preloadedState: any,
```

Comment:
We can write the type as `ReturnType<typeof rootReducer>`

Reply:
Updated.

Putting some other finishing touches on here. I'll update this to be non-draft shortly.